lib/Models/Leaflet.js

```diff
@@ -201,6 +201,28 @@ Leaflet.prototype.computePositionOnScreen = function(position, result) {
     return result;
 };
 
+/**
+ * Add and remove attribution text.
+ * @param {attribution} the attribution object of each Catalog item.
+ */
+Leaflet.prototype.addAttribution = function(attribution){
+    this.map.attributionControl.addAttribution(leafletCredit(attribution));
+};
+
+Leaflet.prototype.removeAttribution = function(attribution){
+    this.map.attributionControl.removeAttribution(leafletCredit(attribution));
+};
+
+/**
+ * A convenient function for handling leaflet credit display
+ * @param {attribution object} for leaflet to display as text or link
+ * @return {[String] text }
```

Comment:
This is generating a jsdoc error.  It should be something like `@return {String} The HTML for the credit.`.